src/subscript/welltest_dpds/welltest_dpds.py

```diff
@@ -354,6 +362,44 @@ def to_csv(filen, field_list, header_list, start=0, end=None, sep=", "):
     print("Writing file:" + filen)
 
 
+def genobs_vec(filen, vec, time):
```

Comment:
`filen` sounds Norwegian in my head.. `filename` too long to type?

Reply:
My private standard used all over, encoded in my muscle memory. The alternative would have to be file_name to comply with PEP8 snake style, and that is just too much typing